From 14804c2f7904d7bf8b0aa5c09c277ad5c74b51ff Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 8 Mar 2024 14:45:23 +0100
Subject: [PATCH 542/841] refactor(stm32mp25-fdts): rename IOM EVAL pins muxing
 configurations

Rename IOM EVAL pins muxing configurations to distinguish
the OSPI instances and the OSPI ports.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git db877908fd33d8914cc38faff8102adfb6558040 refactor(stm32mp25-fdts): rename IOM EVAL pins muxing configurations]
Change-Id: I3685bbe84c763a05f7fb0d23ed3c03720847cac0
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/366685
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp25-pinctrl.dtsi | 6 +++---
 fdts/stm32mp257f-ev1.dts    | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/fdts/stm32mp25-pinctrl.dtsi b/fdts/stm32mp25-pinctrl.dtsi
index afe837576..2e8154470 100644
--- a/fdts/stm32mp25-pinctrl.dtsi
+++ b/fdts/stm32mp25-pinctrl.dtsi
@@ -18,7 +18,7 @@
 	};
 
 	/omit-if-no-ref/
-	ospi1_clk_pins_a: ospi1-clk-0 {
+	ospi_port1_clk_pins_a: ospi-port1-clk-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 0, AF10)>; /* OSPI1_CLK */
 			bias-disable;
@@ -28,7 +28,7 @@
 	};
 
 	/omit-if-no-ref/
-	ospi1_cs0_pins_a: ospi1-cs0-0 {
+	ospi_port1_cs0_pins_a: ospi-port1-cs0-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 3, AF10)>; /* OSPI_NCS0 */
 			bias-pull-up;
@@ -38,7 +38,7 @@
 	};
 
 	/omit-if-no-ref/
-	ospi1_io03_pins_a: ospi1-io03-0 {
+	ospi_port1_io03_pins_a: ospi-port1-io03-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 4, AF10)>, /* OSPI_IO0 */
 				 <STM32_PINMUX('D', 5, AF10)>, /* OSPI_IO1 */
diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 17363f28c..66eac6ff6 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -163,9 +163,9 @@
 
 &ommanager {
 	pinctrl-names = "default";
-	pinctrl-0 = <&ospi1_clk_pins_a
-		     &ospi1_io03_pins_a
-		     &ospi1_cs0_pins_a>;
+	pinctrl-0 = <&ospi_port1_clk_pins_a
+		     &ospi_port1_io03_pins_a
+		     &ospi_port1_cs0_pins_a>;
 	status = "okay";
 
 	spi@40430000 {
-- 
2.47.2

